scripts/build.js

```diff
@@ -69,7 +69,7 @@ function buildFile(file, silent) {
   const relativeToSrcPath = path.relative(packageSrcPath, file);
   const destPath = path.resolve(packageBuildPath, relativeToSrcPath);
 
-  spawnSync('mkdir', ['-p', path.dirname(destPath)]);
+  mkdir.sync(path.dirname(destPath));
```

Comment:
Yes 😅 Since mkdirp was already in jest's dependencies I figured it wouldn't hurt.